sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs

```diff
@@ -194,8 +194,8 @@ public void CanCreateRegisterRequestForUnknownFormatType()
             var format = new SchemaFormat("UnknownType");
             Assert.That(
                 async () => await client.RegisterSchemaAsync(groupName, schemaName, "Hello", format),
-                Throws.InstanceOf<RequestFailedException>().And.Property(nameof(RequestFailedException.Status)).EqualTo(403)
-                    .And.Property(nameof(RequestFailedException.ErrorCode)).EqualTo("NotAvaliable"));
+                Throws.InstanceOf<RequestFailedException>().And.Property(nameof(RequestFailedException.Status)).EqualTo(415)
```

Comment:
This change is to a beta, right?   Otherwise, that'd be breaking, wouldn't it?

Reply:
Yep!

Reply:
(this is to a beta)